packages/docs/content/components/alert.mdx

```diff
@@ -22,27 +28,51 @@ medicare:
 
 ### Error
 
+<ThemeContent onlyThemes={['healthcare']}>
+
+Informs users of an error or critical failure. Often used to block them from proceeding until they resolve the issue including system alerts.
+
+</ThemeContent>
+
 <StorybookExample
   componentName="error alert"
   storyId="components-alert--error-alert"
 />
 
 ### Warning
 
+<ThemeContent onlyThemes={['healthcare']}>
+
+Informs users that their entry or selection may have unexpected or undesirable results, some of which may not be easily undone. Highlights information a user should be aware of in order to help them avoid taking an action that is likely to result in a negative outcome or understand when content/data is missing from the page.
+
+</ThemeContent>
+
 <StorybookExample
   componentName="warning alert"
   storyId="components-alert--warning-alert"
 />
 
 ### Success
 
+<ThemeContent onlyThemes={['healthcare']}>
+
+Confirms the successful completion of a step or task, or receipt of information provided by the user. Suggests a path forward that may be beneficial to the user, based on their selections or entries. 
```

Comment:
Some of this seems like generally useful guidance. Do we have a process in mind for adopting theme-specific guidance into the larger design system?

Reply:
Yeah I was thinking about this too and I think these are cases we can bring up with Melissa to talk about elevating the guidance